### ISSUE TITLE
Switching ports configuration doesn’t work when /usr/ports is a ZFS mount point

### ISSUE THREAD

**Issue by djfiander** [outpaddling/desktop-installer#6]
On my system /usr/ports is a separate ZFS file system, so trying to move it out of the way to switch from current to quarterly fails.

**Comment by outpaddling** [comment]
What version of desktop-installer and auto-admin are you running?  Please send the output of the following:

`pkg info auto-admin desktop-installer`

**Comment by djfiander** [comment]
auto-admin-0.6.10
desktop-installer-0.6.19

These are the versions that were installed when I ran "pkg install desktop-installer" on a fresh 12.2 install.

**Comment by outpaddling** [comment]
That's what I suspected.  The ZFS issue has been resolved in newer versions.  You can do one of the following:
1. Update to latest packages by running auto-pkg-latest or choosing the option in desktop-installer
2. Wait for the 2021Q1 package set, which will contain the fix in 0.7.1.  A standard pkg upgrade should take care of it in the next few days.  I'll be committing even newer versions to both latest and 2020Q1 once the cut-over to git is complete.
3. If you want to live on the edge, you can also run the work-in-progress version:

```
auto-freebsd-wip-checkout
wip-reinstall-port auto-admin
wip-reinstall-port desktop-installer
```

**Comment by djfiander** [comment]
Great. Thanks!
